src/core/plugins/core-commands/server/commands/moderator/inventory.ts

```diff
@@ -41,19 +41,38 @@ class InventoryCommands {
 
     @command('getitem', LocaleController.get(LOCALE_KEYS.COMMAND_GET_ITEM, '/getitem'), PERMISSIONS.ADMIN)
     private static async handleGetItem(player: alt.Player, ...args) {
-        const fullItemName = args.join(' ');
+        if (args.length === 0) {
+            return Athena.player.emit.message(player, LocaleController.get(LOCALE_KEYS.ITEM_ARGUMENTS_MISSING));
+        }
+
+        if (args.length === 1) {
```

Comment:
I think this if statement can have way reduced complexity.

Here's a TypeScript example.
https://www.typescriptlang.org/play?#code/MYewdgzgLgBAhgJwOYRgXhgbQOQAsCmANoSNgDQzYDuIChAJuZVPtE9gKzYC6A3ALAAoIYXyw4AWxABXMLAwBGAYICWAMxgAKRCgB0osEii4YAPgwAmAJQwA3kJiOYoSLAAOIFnJVxCAQSlZeRg3RAh8AEk5bWQITB0IfXxDYxgAWhgFbitlJxh1LQBCFQgAOThSzQ8vKB9-QLkrG3tBPLzJGTl0EM9k2t8AzqhctscE3Q83TRyHJwBfIQXhQRdofJYJcol8bvGAKxAVME1sSisgA